core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -373,6 +373,11 @@ public enum Property {
       "1.10.0"),
   MANAGER_SPLIT_WORKER_THREADS("manager.split.inspection.threadpool.size", "8", PropertyType.COUNT,
       "The number of threads used to inspect tablets files to find split points.", "4.0.0"),
+
+  // This property is a stopgap. See GitHub Accumulo issue #3635.
```

Comment:
I think we can remove this comment everywhere in this PR and the Experimental annotation. The property is addressing a TODO in CompactionJobQueues.

Reply:
Should I also remove the original TODO line in `CompactionJobQueues`?

Reply:
Yes, this is resolving the TODO, right?

Reply:
It's resolving it 50% of the way. @keith-turner and I discussed it and the goal is to have the queue maxSize be configurable on a per-queue basis. #3635 is for resolving the TODO completely.

Reply:
...Now after commenting that, the TODO and comments are just there to ensure that we don't forgot to do the work. 
Since #3635 exists and is assigned I think that accomplishes the same purpose.